packages/fast-components-react-base/src/viewport-positioner/viewport-positioner.tsx

```diff
@@ -879,6 +904,44 @@ class ViewportPositioner extends Foundation<
         ) as ViewportPositionerState);
     };
 
+    /**
+     * Get width of positioner for next render
+     */
+    private getNextPositionerWidth = (
+        desiredHorizontalPosition: ViewportPositionerHorizontalPositionLabel
+    ): number => {
+        let newPositionerRectWidth: number = this.positionerRect.width;
```

Comment:
You could make this less verbose by changing to the following:
```
return this.props.scaleToFit
? Math.min(this.viewportRect.width, this.getAvailableWidth(
    desiredHorizontalPosition
))
: this.positionerRect.width;
```

Reply:
refactored those functions, but used min in the updated version